_includes/coffee/test.coffee

```diff
@@ -329,6 +329,30 @@ describe('test map_undef', ->
 )
 
 
+describe('test github_to_raw', ->
+  it('with github.com address', ->
+    slug = ->
+      'jah5759/benchmark_results'
+    path = ->
+      '1_spinodal_decomp/1a_square_periodic/1a_square_periodic_out.csv'
+    assert.deepEqual(
+      github_to_raw(
+        "https://github.com/#{slug()}/blob/benchmark_results/#{path()}"
+      )
+      "https://raw.githubusercontent.com/#{slug()}/benchmark_results/#{path()}"
+    )
+  )
+  it('without github.com address', ->
+    assert.deepEqual(
+      github_to_raw(
+        'https://bitly.com/'
```

Comment:
This test seems too easy, but robust testing is probably premature optimization. (?)

Reply:
I added more robust testing using the options you suggested.

BTW This has nothing to do with premature optimization. That's when you focus on optimizing a small part of the code without sufficiently understanding the efficiency of the overall problem. This is more a case of not over designing / agile development of the tests before bugs and problems have been identified.